app/controllers/kudos_controller.rb

```diff
@@ -12,7 +12,12 @@ def new
     @kudo = Kudo.new
   end
 
-  def edit; end
+  def edit
+    return unless current_employee != @kudo.giver
+
+    flash[:alert] = 'You are not authorized to perform this operation'
+    redirect_to kudos_path
+  end
```

Comment:
this looks weird

i would do something like 

```
if current_employee != @kudo giver
  flash[:alert] = 'not auth'
  return redirect_to kudos_path
end
```

because it is easier to read what is happening, remember that we are writing code for ourselfs, because we are reading code 10x often than writing it, so we need to write as easy to read code as possible

also you can read this article to get to know locals as a bonus :)
https://www.visuality.pl/posts/should-i-use-instance-variables-in-rails-views

Reply:
Right, this version looks definitely more readable and understandable. I thought about that in the first place, but rubocop suggested a different solution.